tests/integration/defs/accuracy/test_disaggregated_serving.py

```
@@ -25,6 +25,69 @@
 from .accuracy_core import (GSM8K, MMLU, JsonModeEval,
                             LlmapiAccuracyTestHarness, get_accuracy_task)
 
+MAX_PERF_METRICS_REQUESTS = 100
+
+
+def get_worker_env_vars(kv_cache_perf_dir: str = None):
```

Ruff (0.14.5)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)